src/main/java/redis/clients/jedis/BinaryClient.java

```diff
@@ -1405,9 +1406,17 @@ public void hincrByFloat(final byte[] key, final byte[] field, final double incr
   }
 
   public void scan(final byte[] cursor, final ScanParams params) {
+    scan(cursor, params, (byte[]) null);
+  }
+
+  public void scan(final byte[] cursor, final ScanParams params, final byte[] type) {
```

Comment:
Why not put `TYPE` in ScanParams? Just like `MATCH` and `COUNT`

Reply:
@yangbodong22011 TYPE is supported only in SCAN command but not in the other variants. But ScanParams is used in all the variants.

Reply:
@sazzad16 ohh, i got it.